hardware.inc

```diff
@@ -531,15 +532,20 @@ rWY EQU $FF4A
 ; -- Window X Position (R/W)
 ; --
 ; -- 7 <= WX <= 166
+; -- When WX = 7, the window is displayed from the left edge of the LCD screen.
+; -- Values of 0-6 and 166 are unreliable due to hardware bugs.
 ; --
 rWX EQU $FF4B
 
 
 ; --
-; -- KEY 1 ($FF4D)
+; -- SPEED ($FF4D)
 ; -- Select CPU Speed (R/W)
 ; --
-rKEY1 EQU $FF4D

```

Comment:
Don't remove the old name, previous code still uses it.

Reply:
Which one should the bit flags be against?

Reply:
The old one imo.